src/lib/features/feature-search/feature-search-service.ts

```diff
@@ -21,15 +25,48 @@ export class FeatureSearchService {
     }
 
     async search(params: IFeatureSearchParams) {
+        const queryParams = this.convertToQueryParams(params);
         const { features, total } =
-            await this.featureStrategiesStore.searchFeatures({
-                ...params,
-                limit: params.limit,
-            });
+            await this.featureStrategiesStore.searchFeatures(
+                {
+                    ...params,
+                    limit: params.limit,
+                },
+                queryParams,
```

Comment:
Currently expanding with new parameter, but eventually the top params will be removed.